code/src/pages/cart/components/empty-cart/EmptyCart.tsx

```diff
@@ -0,0 +1,38 @@
+import PageWrapper from "@/layouts/page-wrapper/PageWrapper";
+
+import AppBox from "@/components/app-box/AppBox";
+import AppIconButton from "@/components/app-icon-button/AppIconButton";
+import AppLink from "@/components/app-link/AppLink";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+import cartIconWithPlus from "@/assets/icons/cart-with-plus.svg";
+import routes from "@/constants/routes";
+
+import "@/pages/cart/components/empty-cart/EmptyCart.scss";
+
+const EmptyCart = () => {
+  return (
+    <PageWrapper>
+      <AppBox className="empty-cart">
+        <AppTypography
+          className="empty-cart__title"
+          variant="h3"
+          translationKey="cartEmpty.label"
+          data-testid="emptyCartLabel"
+        />
+        <AppTypography translationKey="cartEmpty.subtitle" />
+        <AppIconButton
+          className="empty-cart__img"
+          component={AppLink}
+          to={routes.products.path}
+        >
+          <svg>
+            <use href={`${cartIconWithPlus}#cart-with-plus`} />
+          </svg>
+        </AppIconButton>
```

Comment:
maybe to add `disableRipple` prop from `MUI` for `AppIconButton`
to avoid maybe some click behaviour or smth like that

Reply:
Thank you for this advice. I also thinked about that.
Added !